cypress/support/page/common/selectRoom.js

```diff
@@ -1,10 +1,7 @@
 class e2e_Common_Room {
-    
+
```

Comment:
Already explained above.

Reply:
I'd leave that one to Ankush, since it was created by him and I just made a small change.